### DISCUSSION TITLE
When I use Let'sEncrypt - Internal Error

### DISCUSSION THREAD

**Discussion by BugZau** [jlesage/docker-nginx-proxy-manager#316] [Q&A]
Hi, I follow your guide on YouTube for reverse proxy with Nginx Proxy Manager and I can't get the certification from Let'sEncrypt. When I look at the status of my proxy host, it said online but when I try to make the ssl certificate it said internal error. I'm new with this and hope someone can help me with that. PS I don't have the Let'sEncrypt log in my tmp.

![Screenshot 2024-01-20 121612](https://github.com/jlesage/docker-nginx-proxy-manager/assets/139302712/40d2411a-dd40-4d15-afec-3487a2f36439)
![Screenshot 2024-01-20 121713](https://github.com/jlesage/docker-nginx-proxy-manager/assets/139302712/50c2572e-1dbd-4369-8ad7-f02b056d71b0)

**Comment by jlesage** [comment]
> PS I don't have the Let'sEncrypt log in my tmp.

It should be there.   Did you check *inside* the container ?
```
docker exec <container name> cat /tmp/letsencrypt-log/letsencrypt.log
```
